packages/block-library/src/button/deprecated.js

```diff
@@ -41,6 +41,99 @@ const blockAttributes = {
 };
 
 const deprecated = [
+	{
```

Comment:
It'd be great if a fixture could be added for this new deprecation.

Reply:
Actually, I was wondering if a deprecation is required. It seems as though the existing test fixture didn't fail.

Reply:
Hi @talldan, you are right the existing test fixture did not fail that was the reason we used the isEligible mechanism, but deprecation ensures we migrate the previous style to a new style with a border-radius 0.

Reply:
A fixture was added as suggested 👍